src/main/java/com/exactpro/th2/act/framework/builders/GetActiveWindowBuilder.java

```diff
@@ -48,11 +54,19 @@ protected String getActionName() {
 
 	@Override
 	protected RhAction buildAction() throws UIFrameworkBuildingException {
-		this.checkRequiredFields(windowName, WINDOW_NAME_FIELD_NAME);
+		try
+		{
+			checkRequiredFields(accessibilityId, ACCESSIBILITY_ID_FILED_NAME);
```

Comment:
Looks like you can check ACCESSIBILITY_ID_FILED_NAME field presence and, if it is absent, check WINDOW_NAME_FIELD_NAME as a required one. I.e. there wont be an exception if ACCESSIBILITY_ID_FILED_NAME is absent.